### ISSUE TITLE
WIP Refactor request: more informative variable/parameter/function names

### ISSUE THREAD

**Issue by SKalt** [djrobstep/migra#115]
Working on #114, I've found myself wishing for more informative variable names.  Here's a list of suggested refactors (with ongoing edits):
- changes.py:
  - statements_for_changes
    - things_from, things_target -> source_<type of source>, target_<type of target>
  - statements_from_differences
    - v -> what v is

**Comment by djrobstep** [comment]
Yeah, these admittedly aren't brilliant, but until recently have been for internal consumption only.

Need to think carefully about the `before -> after` naming in particular. Definitely want to standardize but I've had people complain before that the names I chose for these in various places were confusing to them.

**Comment by SKalt** [comment]
> Yeah, these admittedly aren't brilliant, but until recently have been for internal consumption only.

completely understandable.  

> Need to think carefully about the `before -> after` naming in particular. Definitely want to standardize but I've had people complain before that the names I chose for these in various places were confusing to them.

No rush.  I'm glad you got time to look at this issue!
